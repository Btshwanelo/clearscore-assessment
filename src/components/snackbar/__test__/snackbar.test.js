import React from "react";
import Enzyme from "enzyme";
import { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

import Snackbar from "../index";

Enzyme.configure({ adapter: new Adapter() });

describe("snackbar component", () => {
  it("should render correctly", () => {
    const component = shallow(<Snackbar errorMessage="" open={true} />);

    expect(component.debug()).toMatchSnapshot();
  });
});
