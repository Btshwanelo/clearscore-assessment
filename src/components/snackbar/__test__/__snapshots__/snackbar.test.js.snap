// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snackbar component should render correctly 1`] = `
"<ForwardRef(Stack) spacing={2} sx={{...}}>
  <ForwardRef(Snackbar) open=\\"\\" anchorOrigin={{...}} autoHideDuration={6000} onClose={[Function: handleClose]}>
    <ForwardRef(Alert) severity=\\"error\\" sx={{...}} />
  </ForwardRef(Snackbar)>
</ForwardRef(Stack)>"
`;
